### DISCUSSION TITLE
I want to set color for icon

### DISCUSSION THREAD

**Discussion by xuyastc** [nvim-tree/nvim-tree.lua#1775] [Q&A]
Hi there
I want to set the color for the icon，What should I do ?
<img width="702" alt="image" src="https://user-images.githubusercontent.com/37319368/203514183-46a79617-6f82-4894-b215-e71e54ce8200.png">

**Comment by baahrens** [comment]
You can set `NvimTreeIndentMarker`. That will also color the indent lines, though.
